.github/workflows/merge-conflicts.yml

```
@@ -0,0 +1,15 @@
+name: "Merge Conflicts"
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    if: github.repository == 'jellyfin/jellyfin-androidtv'
+    steps:
+      - uses: mschilde/auto-label-merge-conflicts@master
```

```suggestion
      - uses: mschilde/auto-label-merge-conflicts@v2
```

Always try to use a versioned action to prevent conflicts if they release a new version.